codice/ddf
# DDF-1974 Add PPTX Input Transformer

#### What does this PR do?
Add a pptx (powerpoint (ooxml)) input transformer that uses the original tika input transformer for generating the basic metadata and Apache POI for creating the thumbnail.

#### Who is reviewing it?
@ricklarsen @pklinef @rzwiefel @kcwire @Lambeaux 

#### How should this be tested?
Ingest a PPTX file and then search for it. Confirm that the thumbnail represents the first page of the PPTX document. There are several test documents in ddf/catalog/transformer/catalog-transformer-pptx/src/test/resources.

#### Any background context you want to provide?
#### What are the relevant tickets?

https://codice.atlassian.net/browse/DDF-1974

#### Screenshots (if appropriate)
#### Checklist:
